packages/serialize-request/lib/index.js

```diff
@@ -191,7 +191,14 @@ function createSerializedRequest(properties) {
 	);
 }
 
-module.exports = serializeRequest;
+exports = module.exports = serializeRequest;
+
+// We freeze this object so that we avoid any side-effects
+// introduced by the way Node.js caches modules. If this
+// array is edited within a dependent app, then any changes
+// will apply to _all_ uses of `serializeRequest`. This
+// could cause some weird issues so we lock it down.
+exports.DEFAULT_INCLUDED_HEADERS = Object.freeze([...DEFAULT_INCLUDED_HEADERS]);
```

Comment:
question: actually would just `DEFAULT_HEADERS` make more sense? It's a little more concise and it's true in the context of `serializeRequest` 🤔

Reply:
I think this would be fine personally

Reply:
OK I've stopped second-guessing myself, going for the more explicit `DEFAULT_INCLUDED_HEADERS` and if we want to rename later then we can just alias it 😂